src/elli_http.erl

```diff
@@ -87,8 +87,8 @@ keepalive_loop(Socket, NumRequests, Buffer, Options, Callback) ->
       Callback  :: elli_handler:callback(),
       ConnToken :: {'keep_alive' | 'close', binary()}.
 handle_request(S, PrevB, Opts, {Mod, Args} = Callback) ->
-    {Method, RawPath, V, B0} = get_request(S, PrevB, Opts, Callback),
     t(request_start),
+    {Method, RawPath, V, B0} = get_request(S, PrevB, Opts, Callback),
```

Comment:
This changes moves this line:

```erlang
{Method, RawPath, V, B0} = get_request(S, PrevB, Opts, Callback),
```

... from before to after

```erlang
t(request_start),
```

I just want to be sure that was deliberate and that it's a good decision.